pkg/api/titond_api.go

```diff
@@ -27,3 +28,33 @@ func (t *TitondAPI) CreateNetwork(data *model.Network) *model.Network {
 
 	return result
 }
+
+func (t *TitondAPI) CreateL2Geth(data *model.Resource) {
+	// TODO : deal with DB
+	t.db.CreateResource()
+
+	namespace := "default"
+	createL2Geth(t.k8s, namespace)
+}
+
+func createL2Geth(client *kubernetes.Kubernetes, namespace string) {
+	obj := getObject("l2geth", "configMap")
+	client.CreateConfigMap(namespace, obj)
+
+	obj = getObject("l2geth", "pvc")
+	client.CreatePersistentVolumeClaim(namespace, obj)
+
+	obj = getObject("l2geth", "service")
+	client.CreateService(namespace, obj)
+
+	obj = getObject("l2geth", "statefulset")
+	client.CreateStatefulSet(namespace, obj)
+
+	obj = getObject("l2geth", "ingress")
+	client.CreateIngress(namespace, obj)
+}
+
+func getObject(dir, file string) runtime.Object {
```

Comment:
It should be located in kubernestes folder

Reply:
fix it! thank you